src/routes/+page.svelte

```diff
@@ -33,6 +33,9 @@
       bind:data
       bind:problems
       {jsonSchemaURL}
+      on:change={(event) => {
+        console.log("Received change event.", event.detail);
```

Comment:
Playing with it in Netlify I see a couple of properties returned by the event (`editor` and `envelope`) and the envelope property is returned as _null_ sometimes. Will it return not-null as long as the document is valid or has the user to build the document to create the envelope part?

Reply:
> Will it return not-null as long as the document is valid or has the user to build the document to create the envelope part?

The `envelope` property is non-null when there's an envelope (either from the data passed in from the parent component, or the result of build or validate actions), **and** the editor content is valid JSON.